handler/oauth2/flow_resource_owner_test.go

```diff
@@ -38,31 +38,54 @@ func TestResourceOwnerFlow_HandleTokenEndpointRequest(t *testing.T) {
 			description: "should fail because not responsible",
 			expectErr:   fosite.ErrUnknownRequest,
 			setup: func() {
-				areq.GrantTypes = fosite.Arguments{"123"}
+				areq.EXPECT().GetGrantTypes().Return(fosite.Arguments{""})
```

Comment:
I've reinserted it back into the test for the resource owner flow. Please review the latest changes and let me know if it's ok like this.